db/db_test_util.cc

```diff
@@ -1089,11 +1099,21 @@ std::vector<std::uint64_t> DBTestBase::ListTableFiles(Env* env,
 }
 
 void DBTestBase::VerifyDBFromMap(std::map<std::string, std::string> true_data,
+                                 std::map<std::string, Status> status,
```

Comment:
I think it's a good idea to move this argument to the end and default it to `{}`
So that we don't need to pass it every time we use this function

It's up to you if you want to keep it like this